corrector.rb

```diff
@@ -0,0 +1,8 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize.slice(0..9) unless name.length <= 10
+  end
```

Comment:
- [ ] the `capitalize` method doesn't change the actual value of the name, therefore, the return value will always be without capitalization, please use the exclamation mark after the method name in order to change the value, this will look something like `name. capitalize!` 
- [ ] also in this case if the length of the name is not more than 10 it will not capitalize the name at all. you don't need to write an if statement in this case you can just use the slice method, if the name is not longer than 10 characters it will not change anything.

Reply:
Hi @mikethreels, thank you for your feedback and very good point on the conditional, I didn't think about that at all 🤦‍♀️ I have implemented the changes now and pushed again.